src/invidious/yt_backend/youtube_api.cr

```diff
@@ -638,6 +638,11 @@ module YoutubeAPI
     # Send the POST request
     body = YT_POOL.client() do |client|
       client.post(url, headers: headers, body: data.to_json) do |response|
+        if response.status_code == 200
```

Comment:
```suggestion
        if response.status_code != 200
```

Reply:
Sorry I left the wrong condition for testing the error message 😅